cmd/ndm_daemonset/controller/diskstore.go

```diff
@@ -17,18 +17,21 @@ limitations under the License.
 package controller
 
 import (
+	"context"
+
 	"github.com/golang/glog"
-	apis "github.com/openebs/node-disk-manager/pkg/apis/openebs.io/v1alpha1"
+	apis "github.com/openebs/node-disk-manager/pkg/apis/openebs/v1alpha1"
 	"github.com/openebs/node-disk-manager/pkg/util"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 // CreateDisk creates the Disk resource in etcd
 func (c *Controller) CreateDisk(dr apis.Disk) {
 	drCopy := dr.DeepCopy()
-	cdr, err := c.Clientset.OpenebsV1alpha1().Disks().Create(drCopy)
+	err := c.Clientset.Create(context.TODO(), drCopy)
```

Comment:
What is a TODO? Can we add a comment about this of what is it supposed to do?

Reply:
Run type Client APIs need some sort of context. using context.TODO() is way to pass empty context. More on it can be found here ....https://www.notion.so/context-TODO-vs-context-Background-d5224e27ff724a33a79cb4e03a5eb333

We need to take this Context exercise in all modules of OpenEBS sometime soon, it will be fixed in that context.

Reply:
context.TODO() is the way to pass empty context when application/module is not using any context. This is the operator-sdk way of doing things.